test/MUXControls.Test/MUXControls.Test.Shared.targets

```diff
@@ -37,4 +37,5 @@
   <Import Project="..\..\dev\Interactions\SliderInteraction\InteractionTests\SliderInteraction_InteractionTests.projitems" Label="Shared" Condition="$(FeatureInteractionsEnabled) == 'true'" />
   <Import Project="..\..\dev\TabView\InteractionTests\TabView_InteractionTests.projitems" Label="Shared" Condition="$(FeatureTabViewEnabled) == 'true'" />
   <Import Project="..\..\dev\NumberBox\InteractionTests\NumberBox_InteractionTests.projitems" Label="Shared" Condition="$(FeatureNumberBoxEnabled) == 'true'" />
+  <Import Project="..\..\dev\RadialGradientBrush\InteractionTests\RadialGradientBrush_InteractionTests.projitems" Label="Shared" Condition="$(FeatureRadialGradientBrushEnabled) == 'true'" />
```

Comment:
can you move these 3 about the comment?

Reply:
Done for now, but these get placed here by the [GenerateNewControlProject.ps1](https://github.com/microsoft/microsoft-ui-xaml/blob/master/tools/GenerateNewControlProject.ps1) script - is there a better way we could comment these in some way to avoid having to remember to manually fix the script's output every time a new feature is created?  

Could also update the script, but not sure that would be make sense when the comment seems to be about a temporary state.

Reply:
Added note about this to #1836